src/amber/support/client_reload.cr

```diff
@@ -0,0 +1,94 @@
+require "watcher"
+
+module Amber::Support
+  # Used by `Amber::Pipe::Reload`
+  #
+  # Allow clients browser reloading using WebSockets and file watchers.
+  struct ClientReload
+    WEBSOCKET_PATH = rand(0x10000000).to_s(36)
+
+    SESSIONS = [] of HTTP::WebSocket
+
+    def initialize
+      create_reload_server
+      spawn watch_files
+    end
+
+    private def create_reload_server
+      Amber::WebSockets::Server::Handler.new "/#{WEBSOCKET_PATH}" do |session|
+        SESSIONS << session
+        session.on_close do
+          SESSIONS.delete session
+        end
+      end
+    end
+
+    private def reload_clients(msg)
+      SESSIONS.each do |session|
+        session.@ws.send msg
+      end
+    end
+
+    private def watch_files
+      puts "🔃  Your ClientBot is vigilant. beep-boop..."
+      watch(["public/**/*"]) do |event|
+        event.on_change do |files|
+          files.each do |file, timestamp|
+            puts "🔃  watching file: ./#{file}"
```

Comment:
This should use the new logger `Amber.logger.puts "watching file: ./#{file}", "Watcher`

Reply:
Yeah, I thought you weren't using logger on watcher, I just realize latest amber `v0.5.0` was republished to fix logs. 😅 

Let me fix this 👍

Reply:
Done!